api/opensource/mfx_dispatch/libmfx_vs12.sln

```diff
@@ -0,0 +1,43 @@
+﻿
```

Comment:
You have 2 solutions: this one and libmfx_vs2017.sln. Can you, please, align names and use the same pattern across .sln/.vcxproj?